src/reducers/query.jsx

```diff
@@ -118,6 +125,34 @@ export function changeDateRange( state, action ) {
 }
 
 /**
+* Change a boolean flag filter
+*
+* @param {object} state the current state in the Redux store
+* @param {object} action the payload containing the value to change
+* @returns {object} the new state for the Redux store
+*/
+export function changeFlagFilter( state, action ) {
+
+  /* eslint-disable camelcase */
+  const newState = {
+    ...state,
+    has_narrative: action.filterValue
```

Comment:
This was an accidental relic of when I was making that decision. I have fixed it to use the fieldName now (so that it is extendable later)